CMakeLists.txt

```diff
@@ -1,18 +1,21 @@
 # This is the main CMake file for NCEPLIBS-wrf_io.
 #
-# 
+# Other programmer, Ed Hartnett
 cmake_minimum_required(VERSION 3.15)
 
+# Get the version from the VERSION file.
 file(STRINGS "VERSION" pVersion)
 
-project(
-  wrf_io
-  VERSION ${pVersion}
-  LANGUAGES Fortran)
+# Initialize project.
+project(wrf_io VERSION ${pVersion} LANGUAGES Fortran)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules/Modules")
+# Add custom CMake Modules.
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/"
+  CACHE INTERNAL "Location of our custom CMake modules.")
```

Comment:
CACHE INTERNAL is a potential problem if wrf_io is included as a sub-project.
e.g. say a project does `add_subdirectory(wrf_io)`, this statement with FORCE the CMAKE_MODULE_PATH to be this path.